cmd/webhook-manager/main.go

```diff
@@ -58,6 +58,10 @@ func main() {
 		klog.Fatalf("Configured port is invalid: %v", err)
 	}
 
+	if err := config.ReadCAFiles(); err != nil {
+		klog.Fatalf("Failed to read CA files: %v", err)
+	}
```

Comment:
please clarify what volcano user should do if the cert file is encripted. It's better to add a decrypt func.

Reply:
> 

optimized it. the ParseCAFiles obtain the action to read files and the action to decrypt the data if a decrypt func is as a argument